src/CMakeLists.txt

```diff
@@ -245,15 +196,26 @@ target_include_directories(valhalla
 target_link_libraries(valhalla
   PUBLIC
     ${libvalhalla_link_libraries}
+    $<$<BOOL:$(WIN32)>:ws2_32>
```

Comment:
actually this is only needed in baldr & mjolnir, but I couldn't find a way to make it a proper "target" so it could be used in baldr/CMakeLists.txt. meaning it's probably broken when building static libraries for each module, but that's not smth one would usually do on windows anyways I guess